ci/README.md

```diff
@@ -0,0 +1,202 @@
+# Continuous Integration
+
+When a Pull Request (PR) is opened, several automated checks are performed to ensure the new compiler still builds, passes tests, and does not break external projects.
+Since DMD runs on many platforms and lots of D infrastructure depends on it, they are spread over many CI services.
+
+## Troubleshooting failed checks
+When your PR fails checks, click on the "Details" link next to it to see what the error is.
+
+Errors caused by the changes you made should be addressed by pushing new commits to your branch.
+
+When it looks like the error is unrelated to your PR's changes, try rebasing your branch.
+First, ensure that you have a 'remote' pointing to both dlang/dmd and your own fork of dmd, for example:
+```
+git remote -v
+origin  git@github.com:your-github-name/dmd.git (fetch)
+origin  git@github.com:your-github-name/dmd.git (push)
+upstream        git@github.com:dlang/dmd.git (fetch)
+upstream        git@github.com:dlang/dmd.git (push)
+```
+
+If you don't have an `upstream` remote, you can add it with:
+```
+git remote add upstream git@github.com:dlang/dmd.git
+```
+
+Then, pull the latest changes from the branch that your PR merged into (`master` or `stable`) and rebase your branch.
+```
+git checkout master
+git pull --ff-only upstream master
+git checkout your-branch
```

Comment:
You can just `git fetch` then `git rebase dlang/master`, no?

Reply:
I guess this is just my [workaround](https://xkcd.com/763/)